tests/sles4sap/migrate_sles_to_sles4sap.pm

```diff
@@ -48,8 +52,16 @@ sub run {
     send_key 'ret';
     wait_serial "Please enter your activation code", timeout => 5;
     enter_cmd "${regcode}";
-    assert_script_run "ls /tmp/OK";
-    zypper_call "in -y -t pattern sap_server";
+
+    # test either a failing migration or a working one
+    if ("${regcode}" eq "invalid_key") {
```

Comment:
Perl is not Bash :)

I think just `$regcode` will work.

Reply:
Good point, I just wanted to be consistent with what was already there 3 lines above.

Reply:
I changed both occurrences of the bashism :-)

Reply:
Sometimes "bashisms" are necessary:
```
# perl
$msg = 'Hello ';
print "Without bashisms: $msgWorld\n";
print "With    bashisms: ${msg}World\n";
Without bashisms: 
With    bashisms: Hello World
```
Or when doing some more complex data structures handling:
```
    for my $arg (@{$param_file}) {
        next unless $arg =~ $PARAM_RE;
        push @{$params{$1}}, path($2)->slurp;
    }
```
But agree with @ricardobranco777 .... it does look better without the curly braces. :)